clients/fides-js/__tests__/lib/gpp/us-notices.ts

```diff
@@ -6,11 +6,11 @@ import {
   setGppOptOutsFromCookie,
 } from "../../../src/lib/gpp/us-notices";
 import {
+  FidesCookie,
   PrivacyExperience,
   PrivacyNotice,
 } from "../../../src/lib/consent-types";
 import { makeStub } from "../../../src/lib/gpp/stub";
-import { FidesCookie } from "../../../src/lib/cookie";
```

Comment:
if you're having problems with circular dependencies on importing types, you can normally do 

```ts
import type { FidesCookie } from "../../../src/lib/cookie";
```
instead

Reply:
ohhh lovely! Thanks so much!